CHANGELOG.md.orig

```diff
@@ -0,0 +1,220 @@
+# Changelog
```

Comment:
backup file was added to the repo. I think it could be useful for you to add `*.orig` pattern to you `.global_gitignore` file